packages/manager/src/utilities/formatRegion.ts

```diff
@@ -4,7 +4,14 @@ import {
 } from '@linode/api-v4';
 import { Region } from '@linode/api-v4';
 
-import type { Country } from '@linode/api-v4';
+import type { Agreements, Country, Profile } from '@linode/api-v4';
+
+interface GDPRConfiguration {
+  agreements?: Agreements;
+  profile?: Profile;
```

Comment:
Should we switch to 
```suggestion
  agreements: Agreements | undefined;
  profile: Profile | undefined;
```

so that profile and agreements are required to be passed even though they may be undefined?